configuration/customize-dashboard.md

```diff
@@ -0,0 +1,38 @@
+---
+description:  This article describes how you can customize your Dashboard by rearranging the boxes. 
+---
+
+# Syskit Point Dashboard
+
+The **Syskit Point** Dashboard shows administrators the following information in boxes:
+ * Inventory
+ * Users
+ * Storage
+ * Security & Compliance
+ * External Collaboration
+ * Active Access Review
+ * Lifecycle Management
+ * Microsoft 365 Licenses
+
+The **Syskit Point** Dashboard shows collaborators the following information in boxes:
```

Comment:
isto ko gore pošto su to baš naše Point role, ovdje možda napisati Syskit Point Collaborators.